### ISSUE TITLE
Sort only selected symbols

### ISSUE THREAD

**Issue by SofieHaugeKatan** [sonburn/symbol-organizer#15]
Sometime it's would be really useful to choose only a subset of symbols to organize. 
Would it be possible to ad a checkbox for only sorting selected symbols?

**Comment by sonburn** [comment]
Thank you for the feedback, I am glad the plugin is useful for you! Your suggestion is something I can consider for a future enhancement, but I do have some questions.

Is there a particular reason or use case where you would not want just click one button to organize everything, vs manually selecting a subset of symbols and additional checkboxes within the plugin?

Considering the sort moves things around on the symbol canvas, I am not sure how I would be able to achieve this without at least _moving_ the unselected symbols to make space, and I imagine it would at least be useful to display those unsorted symbols in a new grid (to remove any gaps left from the selected symbols)?

**Comment by geoffb77** [comment]
Agree, great plugin! as you mention not sure if possible, but naming conventions can be different for different symbol types. So ability to sort selected symbols by group definition would be useful. eg i may want to sort all my text symbols by /2nd/ match and say my icon symbols by /1st/ match
